.idea/.gitignore

```diff
@@ -0,0 +1,12 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+vcs.xml
+modules.xml
+go-dsa.iml
```

Comment:
Can we remove the extra ones from here? I think only the last line is relevant to this project.